### DISCUSSION TITLE
Which Splunk Version's are supported from which SOK Version?

### DISCUSSION THREAD

**Discussion by markusspitzli** [splunk/splunk-operator#1075] [Q&A] [answered]
Today I received an email with the latest Security alerts for the Splunk products. These alerts are also documented on Splunks [advisory page](https://advisory.splunk.com/?301=%2Fen_us%2Fproduct-security.html&LST=Email1).

Patching means that we have to install the latest version, at least minor version, of Splunk Enterprise. 

I checked on docker hub and Splunk did actually release [the latest version](https://hub.docker.com/r/splunk/splunk/tags) also as a docker image. That means an upgrade would be possible. 

The Splunk Operator supports certain [Splunk Enterprise Versions](https://github.com/splunk/splunk-operator/blob/main/docs/README.md#docker-requirements). Up to a certain point, I just can upgrade Splunk Enterprise without upgrading the Splunk operator. 

What if I have an older SOK version of in place. like 2.1.1 or below. Is there an overview which SOK Version does support which Splunk Enterprise version? And at what version isn't it supported anymore?
In the end, a table would very helpful which states Which Splunk Version's are supported from which SOK Version.

**Comment by vivekr-splunk** [comment, marked as answer]
@markusspitzli thanks for your question here. we are still working to fix CVE issues on Splunk docker image. we will update once they are fixed.

**Reply by markusspitzli** [reply]
@vivekr-splunk thanks for you answer. I assumed that the docker image of Splunk Enterprise 9.0.4 contains the latest CVE. Is this a wrong assumption?

**Comment by amacint1** [comment]
Looking for help clarifying when SOK 3.1 and the Splunk standalone images will officially support Kubernetes 1.35? We are currently upgrading from 1.33 to 1.34 and are already planning our next upgrade cycle.
